### ISSUE TITLE
Use Fahrenheit where appropriate

### ISSUE THREAD

**Issue by nu8** [chubin/wttr.in#481]
wttr.in knows where I am:

~~~
PS D:\Desktop> curl.exe wttr.in
Weather report: Irving, United States of America
~~~

So in turn, it knows that I **dont** use Celcius, unless I am a scientist
(Im not). So it seems strange that I have to pass `?u` to get what it already
knows that I want.

**Comment by chubin** [comment]
It is a bug; actually it should use Fahrenheit for queries from the US.
(and even did it partially, but obviously not everywhere; bug).

I believe, it works well for PNG queries.
Could you check if it is Fahrenheit here?

https://wttr.in/Nuremberg.png

(the PNG is rendered just in time for each query)

**Comment by nu8** [comment]
![Nuremberg](https://user-images.githubusercontent.com/61808913/85857818-2e609580-b780-11ea-9cf6-7684c46e5ea8.png)

**Comment by jsatk** [comment]
I too have noticed this suddenly.

**Comment by chubin** [comment]
Ok, then it is a bug; I acknowledge it; I hope to  fix it soon

**Comment by imnisp** [comment]
I cannot get farenheit units when saving output as png despite my separating the options with an underscore. Here is the commandline that does *NOT* work:
curl -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0' -s --output forecast.png wttr.in/38.8977,77.0365_u.png

If I output to the terminal (not as PNG) and change the underscore to a questionmark, output *IS* in Fahrenheit.

**Comment by chubin** [comment]
@imnisp Yes, I acknowledge this. It is a bug. If you use `?u` instead of `_u` it will work:
```
wttr.in/38.8977,77.0365.png?u
```
but it is, of course, a bug anyway and must be fixed
